todo/src/reducer/toDo.js

```diff
@@ -0,0 +1,15 @@
+import {ADD ITEM, TOOGLE ITEM} from 
+
+const toDo = (state =[], actions) => {
+    switch(action.type) {
+        case 'ADD ITEM':
+        return state.concat(action.payload);
+        case 'TOOGLE ITEM':
+        return state.map(item => item !== action.payload);
+        default:
+        return state;
+
+    }
```

Comment:
You're on the right track here but I'm not sure your returns will return what you want. For add item, I'm not seeing a new item before you concat it? Check out https://redux.js.org/recipes/structuring-reducers/refactoring-reducers-example for a little insight.

You'll also need to finish your import statement or it won't be connected to the rest of the app.